surfman/src/platform/egl/surface/mod.rs

```diff
@@ -0,0 +1,87 @@
+// surfman/surfman/src/platform/egl/android_surface.rs
```

Comment:
I think this file name is incorrect now. To be honest, I'm not sure why the file name is included at the top of these files. It seems redundant.

Reply:
Yeah, I was also wondering about this, but assumed that perhaps there was some kind of tooling I don't know about, which requires these comments. 

I will fix the name here, but if there is no reason for these comments (anymore), then I'd also be happy to open a PR that removes all of the filenames at the top.